src/scenes/ApplicationRouter/portal/Review/index.tsx

```diff
@@ -88,12 +122,20 @@ const ReviewView = () => {
     [applicationSections]
   );
 
-  return (
-    <ApplicationPage crumbs={crumbs}>
-      {selectedApplication?.status === 'In Review' && <ApplicationStatusSubmitted />}
-      {selectedApplication?.status !== 'Submitted' && <ReviewCard sections={nonPrescreenSections} />}
-    </ApplicationPage>
-  );
+  const renderContent = (application: Application | undefined) => {
+    switch (application?.status) {
+      case 'In Review':
```

Comment:
Similarly we need to add all of these for the in review cases:

Submitted
PL Review
Ready for Review
Pre-check
Needs follow-up
Carbon Eligible